app/src/main/java/com/example/puzzlepic/ui/main/MainFragment.kt

```diff
@@ -29,15 +29,8 @@ class MainFragment : Fragment() {
     private val IMAGE_GALLERY_REQUEST_CODE: Int = 2001
     private val SAVE_IMAGE_REQUEST_CODE: Int = 1999
     private val CAMERA_REQUEST_CODE: Int = 1998
-
-    val CAMERA_PERMISSION_REQUEST_CODE = 1997
-
+    private val CAMERA_PERMISSION_REQUEST_CODE = 1997
```

Comment:
Adding a visability modifier and doing soem housekeeping in the variable section to make it neat. Reference: https://kotlinlang.org/docs/visibility-modifiers.html#classes-and-interfaces